src/Illuminate/Cache/composer.json

```diff
@@ -18,7 +18,8 @@
         "illuminate/collections": "^11.0",
         "illuminate/contracts": "^11.0",
         "illuminate/macroable": "^11.0",
-        "illuminate/support": "^11.0"
+        "illuminate/support": "^11.0",
+        "psr/simple-cache": "^1.0|^2.0|^3.0"
```

Comment:
This isn't correct. This package doesn't uses this dependency directly but through a Symfony component. Please revert this.

Reply:
Oh, my bad. `illuminate\cache` doesn't use directly `psr/simple-cache`, it uses through `illuminate/contracts`. I restored this change and updated PR's description. Thank you!
https://github.com/laravel/framework/blob/11.x/src/Illuminate/Contracts/Cache/Repository.php

```php
<?php

namespace Illuminate\Contracts\Cache;

use Closure;
use Psr\SimpleCache\CacheInterface;

interface Repository extends CacheInterface
{
    //
}
```